pom.xml

```diff
@@ -154,30 +166,12 @@
                 <artifactId>cobertura-maven-plugin</artifactId>
                 <version>2.4</version>
             </plugin>-->
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>findbugs-maven-plugin</artifactId>
-                <version>2.0.1</version>
-                <configuration>
-                    <xmlOutput>true</xmlOutput>
-                    <!-- Optional derectory to put findbugs xdoc xml report -->
-                    <!--<xmlOutputDirectory>target/site</xmlOutputDirectory>-->
-                </configuration>
-            </plugin>
         </plugins>
     </reporting>
-    <dependencies>
-        <dependency>
-            <groupId>com.google.guava</groupId>
-            <artifactId>guava</artifactId>
-            <version>12.0-rc1</version>
-            <scope>provided</scope>
-        </dependency>
-    </dependencies>
```

Comment:
com.google.guava:guava:jar:11.0.1 is provided via Jenkins core, and rebuild-plugin doesn't seem require features of 12.0-rc1.